perllib/FixMyStreet/Cobrand/Oxfordshire.pm

```diff
@@ -53,6 +53,27 @@ sub users_can_hide { return 1; }
 
 sub default_show_name { 0 }
 
+sub lookup_by_ref_regex {
+    return qr/^\s*((?:ENQ)?\d+)\s*$/;
+}
+
+sub lookup_by_ref {
+    my ($self, $ref) = @_;
+
+    my $criteria = [
+        id => $ref,
+        external_id => $ref
+    ];
+
+    if ( $ref =~ /^ENQ/ ) {
+        my $filter = "%T18:customer_reference,T\\d+:$ref,%";
+        $criteria = { 'extra' => \[ 'SIMILAR to ?', [ {} => $filter ] ] };
```

Comment:
This could be `{ extra => [ '~', "T18:customer_reference,T\\d+:$ref," ] }`? All `SIMILAR TO` does is rewrite to a regex afaik.